### PR TITLE
:tada:Source hubspot add new streams for engagements(calls, emails, meetings, notes and tasks)

### PR DESCRIPTION
**Pull request by vladimir-remar**
## What
Add new streams for engagements
- Calls
- Emails
- Meetings
- Notes
- Tasks

## How

Add these entries into `client.py` using the `CRMSearchStream` class
## 🚨 User Impact 🚨
Are there any breaking changes? What is the end result perceived by the user? If yes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.

## Pre-merge Checklist
Expand the relevant checklist and delete the others. 

<details><summary> <strong> Updating a connector </strong></summary>
<p>
   
#### Community member or Airbyter
   
- [x] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the new connector version is published, connector version bumped in the seed directory as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

### CODE REVIEW

**Comment by marcosmarxm** [comment]
Thanks @vladimir-remar can you share the output from `./gradlew airbyte-integrations:connectors:source-hubspot:integrationTest`? You can run only the streams you have added. To achieve this you can comment the oauth tests in acceptance-test.yaml file and comment the configured_catalog inside integration_test folder.

**Comment by marcosmarxm** [comment]
And run `./gradlew format`

**Comment by vladimir-remar** [comment]
> Thanks @vladimir-remar can you share the output from `./gradlew airbyte-integrations:connectors:source-hubspot:integrationTest`? You can run only the streams you have added. To achieve this you can comment the oauth tests in acceptance-test.yaml file and comment the configured_catalog inside integration_test folder.

Thanks @marcosmarxm, I attached the log, as you suggest me, I commented all about the oauth and just run the tests over the new streams
[hubspot_gradlew.log](https://github.com/airbytehq/airbyte/files/7941371/hubspot_gradlew.log)

File: docs/integrations/sources/hubspot.md [hunk 1]
```diff
@@ -112,6 +112,7 @@ If you are using Oauth, most of the streams require the appropriate [scopes](htt
 
 | Version | Date       | Pull Request | Subject                                                                                                                                        |
 |:--------|:-----------| :--- |:-----------------------------------------------------------------------------------------------------------------------------------------------|
+| 0.1.37  | 2022-01-25 | [9786](https://github.com/airbytehq/airbyte/pull/9786) | Add new streams for engagemnts(calls, emails, meetings, notes and tasks) |
```
Comment (alafanechere) [inline docs/integrations/sources/hubspot.md:115]:
```suggestion
| 0.1.37  | 2022-01-25 | [9786](https://github.com/airbytehq/airbyte/pull/9786) | Add new streams for engagements(calls, emails, meetings, notes and tasks) |
```

**Review by alafanechere — APPROVED** [review]
The acceptance tests are passing. I made some changes to the changelog and the version number. I'm publishing the connector now.

**Comment by alafanechere** [comment]
Thank you for your contribution @vladimir-remar. The connector is now published 🎉

**Comment by vladimir-remar** [comment]
@alafanechere Thanks!!!

**Pull request merged**